apps/examples/testcase/le_tc/compression/Kconfig

```diff
@@ -16,4 +16,10 @@ config TC_LZMA
 	default y
 	select LIBLZMA
 
+config TC_COMPRESS_READ
```

Comment:
May be this config can be added under a  TC_COMPRESS_ALL config as follows:
if EXAMPLES_TESTCASE_COMPRESSION
config TC_COMPRESS_ALL
        bool "All"
        default y
        select TC_LZMA
        select TC_COMPRESS_READ

config TC_LZMA
....
....
config TC_COMPRESS_READ
bool "Compress_read_testcase"
	default n
	select LIBLZMA
	select COMPRESSED_BINARY
endif

So that all the compression/decompression testcases can be enabled at once.